core/src/main/scala/Block.scala

```diff
@@ -30,6 +30,14 @@ object Block {
       valueLength: Int = 90
   ): Block =
     Block.fromSource(Source.fromURI(path.toURI), keyLength, valueLength)
+
+  def sort(block: Block): Block = {
+    val sortedRecords = block.records.sortBy(_.key)
+    new Block(sortedRecords)
+  }
+  def sample(block: Block): List[Key] = {
```

Comment:
Should be defined in `class Block` with `def sample()`

Reply:
ack